frontend/src/components/TASKS/TaskDetailModal.jsx

```
@@ -0,0 +1,182 @@
+            value={task.status}
+            onChange={(e) => {
+              const newStatus = e.target.value;
+              setTask({ ...task, status: newStatus });
+            }}
+          >
+            <option value="Incomplete">Incomplete</option>
+            <option value="In-Progress">Pending</option>
+            <option value="Completed">Completed</option>
+          </select>
+
+          <button className={styles.actionButton} onClick={handleSave}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)